runelite-client/src/main/java/net/runelite/client/plugins/roguesden/RoguesDenOverlay.java

```diff
@@ -51,6 +59,7 @@ public RoguesDenOverlay(Client client, RoguesDenPlugin plugin)
 		setLayer(OverlayLayer.ABOVE_SCENE);
 		this.client = client;
 		this.plugin = plugin;
+		font = FontManager.getRunescapeFont().deriveFont(Font.BOLD);
```

Comment:
Might want to use the default dynamic overlay font here. I think you can just not set a font and use the current instead.